azure-pipelines.yml

```diff
@@ -106,11 +106,11 @@ jobs:
   timeoutInMinutes: 120
   strategy:
     matrix:
-      java:
-        AZURE_SDK_REPO: azure-sdk-for-java
+      ruby:
```

Comment:
why removed java and python?

Reply:
Yaohai don't want too much auto pr right now, so only enable v2 target test repo. Zim want python sdk to be ran in production, so disable v1.

Reply:
The semantic check in CI failed because of a tooling bug, Ray has already known the issue. So please bypass checks and force merge it.

Reply:
I believe this configuraiton is also used  by v1, right? If you removed  java, java will not be covered by v1 also.